examples/host-device-rdma/10-nicclusterpolicy.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: mellanox.com/v1alpha1
```

Comment:
Do we want the processed yamls in the repo also?
Will it useful for the future planned testing?